integration/Dockerfile

```diff
@@ -1,8 +1,11 @@
 FROM python:3.6
 
+WORKDIR /opt/cook/integration
+COPY requirements.txt /opt/cook/integration
 ADD cli.tar.gz /opt/cook/cli/
+RUN pip install -r requirements.txt
 COPY . /opt/cook/integration
 WORKDIR /opt/cook/integration
```

Comment:
Since `WORKDIR` is now run as the first command, this duplicate line should be deleted.